packages/uiweb/src/lib/components/chat/ChatViewList/ChatViewList.tsx

```diff
@@ -133,15 +141,143 @@ export const ChatViewList: React.FC<IChatViewListProps> = (options: IChatViewLis
     };
   }, [chatId, user]);
 
-  // When loading is done
+  // When loading is done - fetch chat messages
   useEffect(() => {
     if (initialized.loading) return;
 
     (async function () {
-      await getMessagesCall();
+      await fetchChatMessages();
     })();
   }, [initialized.loading]);
 
+  // when chat messages are changed or chat reactions are changed
+  useEffect(() => {
```

Comment:
Why not create a useMessageScroll custom hook in a separate file and move all the scroll related login into it.

Reply:
Since this only ties with ChatViewList, moving all the scroll related functionality can be done at a later stage when and if the functionality needs to be used by other components?

Reply:
Okay